_unittest_solvers/test_26_emit.py

```diff
@@ -694,16 +698,14 @@ def test_12_radio_getters(self, add_app):
     )
     def test_13_static_type_generation(self):
         domain = self.aedtapp.results.interaction_domain()
-        if sys.version_info < (3, 8):
+        if sys.version_info < (3, 10):
             py_version = "EmitApiPython"
-        elif sys.version_info < (3, 9):
-            py_version = "EmitApiPython38"
-        elif sys.version_info < (3, 10):
-            py_version = "EmitApiPython39"
         elif sys.version_info < (3, 11):
             py_version = "EmitApiPython310"
         elif sys.version_info < (3, 12):
             py_version = "EmitApiPython311"
+        elif sys.version_info < (3, 13):
+            py_version = "EmitApiPython312"
```

Comment:
Could we remove those if/elif and use `py_version = f"EmitApiPython{sys.version_info[0]}{sys.version_info[1]}"` instead ?

Reply:
Yeah, thanks for the idea. I also applied this to lines 48-64.